modules/core/Core.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+
+/**
+ * Protocol that represents Core SDK API
+ */
+public protocol AgsCoreServices {
+    /**
+     * Get configuration for specific service reference
+     */
+    func getConfiguration(_ serviceRef: String) -> [Service]
+}
+
+/**
+ * AeroGear Core interface to be used directly by SDK services
+ */
+public class AgsCore: AgsCoreServices {
+    let config: Config
+
+    init() {
```

Comment:
`init(_ config: Config = Config())` ?

Reply:
This needs to be config from json. Cannot be passed to core.